docs/static/openapi.yml

```diff
@@ -26730,6 +26816,26 @@ definitions:
         type: boolean
       extraInfo:
         type: string
+  Pylonstech.pylons.pylons.RecipeHistory:
+    type: object
+    properties:
+      ItemID:
+        type: string
+      CookbookID:
+        type: string
+      RecipeID:
+        type: string
+      Sender:
+        type: string
+      SenderName:
+        type: string
+      Reciever:
```

Comment:
"Reciever" has to be changed to "Receiver" in multiple places.

Reply:
updated